dockstore-webservice/src/main/java/io/openapi/api/impl/ToolsApiServiceImpl.java

```diff
@@ -92,7 +90,7 @@
     private static final String GITHUB_PREFIX = "git@github.com:";
     private static final String BITBUCKET_PREFIX = "git@bitbucket.org:";
     private static final int SEGMENTS_IN_ID = 3;
-    private static final int DEFAULT_PAGE_SIZE = 1000;
+    private static final int DEFAULT_PAGE_SIZE = 100;
```

Comment:
This is also the max page size; name could indicate it. It might make the merge back to develop more difficult though.

Reply:
yeah, I'll code comment for now pending one of those disruptive refactorings